index.html

```diff
@@ -3,6 +3,7 @@
 title: "Multi-node OpenStack"
 description: "MicroStack is an upstream single-node OpenStack deployment which can run directly on your workstation.
```

Comment:
I can't tell what the description is in the copy doc but looks like it could be different fro mthe bottom of the doc?

Reply:
I updated it to: MicroStack is an upstream multi-node OpenStack deployment which can run directly on your workstation. Although made for developers, it is also suitable for edge, IoT and appliances.